### DISCUSSION TITLE
Another question about galaxy brain

### DISCUSSION THREAD

**Discussion by sourabmaity** [Schweinepriester/github-profile-achievements#104] [Q&A]
@tanny99 here is another question
open a discussion for me to complete galaxy brain

**Comment by tanny99** [comment]
Okay

**Comment by foxyseta** [comment]
Can someone mark my answer too? Pleasee

**Comment by thenithinbalaji** [comment]
# How do you get Galaxy brain?
+ Answer at least 2 discussions. 
+ Take note, you cannot mark your own answers to your own questions.

![image](https://user-images.githubusercontent.com/73932121/179235281-69524f35-21a5-4c53-a89d-35dccc87ebf0.png)

# This badge not only has the base version, but also tiers.

![image](https://user-images.githubusercontent.com/73932121/179235882-033cffe0-4bbd-44e2-a3be-b1a078745b05.png)

**Reply by thenithinbalaji** [reply]
@sourabmaity
